code/modules/research/designs/comp_board_designs.dm

```diff
@@ -118,6 +118,16 @@
 	)
 	departmental_flags = DEPARTMENT_BITFLAG_ENGINEERING | DEPARTMENT_BITFLAG_SECURITY //Honestly should have a bridge techfab for this sometime.
 
+/datum/design/board/bankmachine
+	name = "Bank Machine Board"
+	desc = "All0ws for the construction of circuit boards used to build a Bank Machine."
+	id = "bankmachine"
+	build_path = /obj/item/circuitboard/computer/bankmachine
+	category = list(
+		RND_CATEGORY_COMPUTER + RND_SUBCATEGORY_COMPUTER_COMMAND
+	)
+	departmental_flags = DEPARTMENT_BITFLAG_ENGINEERING | DEPARTMENT_BITFLAG_SECURITY
```

Comment:
Comms console is command only, right? Should bank machine be treated the same? Or even be in cargo instead of engineering

Reply:
Yeah, sounds better to it be in cargo lathe.